### ISSUE TITLE
get_tukeyQcrit vs get_tukeyQcrit2?

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#5046]
in sandbox.stats.multicomp (which unfortunately is used outside the sandbox despite having abysmal QA), these functions have identical docstrings.  What is the meaningful difference?  Why is the first used in some contexts and the latter in others?

**Comment by jbrockmendel** [comment]
@josef-pkt thoughts on this?  It looks like the 2 version is intended to fully replace the older one, but this seems like the kind of thing you might want to keep around.

**Comment by josef-pkt** [comment]
The first uses a table of critical values, the second uses critical values based on a better functional approximation, qsturng which included/copied from another python package.
(There is a third version that uses numerical integration, but that is computationally not robust in that version.)
AFAIR, code was left in there for cross-checking. In some other cases of hypothesis test we give users an option whether to use values based on a table or on a approximation formulas.

(multiple comparisons needs at least one, or several, more "rounds" to go beyond the main currently supported tukey-hsd function/class)

**Comment by jbrockmendel** [comment]
> multiple comparisons needs at least one, or several, more "rounds" 

Yep.  There are parts of this (e.g. turning the cross-checking code into actual tests) that non-you contributors can work towards; that should be encouraged.
